packages/plugin-drizzle/src/utils/config.ts

```diff
@@ -67,7 +50,30 @@ const configCache = createContextCache(
 
         return columnName;
       },
-      ...config,
+      getPrimaryKey: (tableName) => {
```

Comment:
this is kinda hacky, would be better if drizzle could expose this properly (also need to test composite primaryKeys actually work across drivers)